joblib/test/test_memory.py

```diff
@@ -479,6 +479,21 @@ def test_persistence(tmpdir):
     gp(1)
 
 
+def test_check_call_in_cache(tmpdir):
+    for func, Result in zip((MemorizedFunc(f, tmpdir.strpath),
```

Comment:
What is the point of the `Result` object here? I don't see it used.

Reply:
Ah yeah, was just copy/pasting. Cleaned that up now.

Reply:
FYI squashed those into one commit now